xbmc/filesystem/IFileTypes.h

```diff
@@ -48,10 +48,10 @@ struct SNativeIoControl
 
 struct SCacheStatus
 {
-  uint64_t forward;  /**< number of bytes cached forward of current position */
-  unsigned maxrate;  /**< maximum number of bytes per second cache is allowed to fill */
-  unsigned currate;  /**< average read rate from source file since last position change */
-  bool     lowspeed; /**< cache low speed condition detected? */
+  uint64_t forward; /**< number of bytes cached forward of current position */
+  unsigned maxrate; /**< maximum allowed read(fill) rate (bytes/second) */
+  unsigned currate; /**< average read rate (bytes/second) since last position change */
+  unsigned lowrate; /**< low speed read rate (bytes/second) (if any, else 0) */
```

Comment:
There seems to be a mixture of `unsigned` and `unsigned int` being used. Do you think we could settle on using a sized type instead like `uint32_t`?

Reply:
Yes sounds good. Obviously this is all legacy stuff.

Reply:
@lrusak : Added a commit with the unsigned -> uint32_t changes